kornia/utils/image.py

```diff
@@ -93,21 +93,24 @@ def _to_bcdhw(tensor: torch.Tensor) -> torch.Tensor:
     return tensor
 
 
-def tensor_to_image(tensor: torch.Tensor) -> "np.ndarray":
+def tensor_to_image(tensor: torch.Tensor, keepdim: bool = False) -> "np.ndarray":
     """Converts a PyTorch tensor image to a numpy image.
 
     In case the tensor is in the GPU, it will be copied back to CPU.
 
     Args:
         tensor: image of the form :math:`(H, W)`, :math:`(C, H, W)` or
             :math:`(B, C, H, W)`.
+        keepdim: If ``False`` squeeze the input image to match the shape
+            :math:`(H, W, C)`.
```

Comment:
we are returning a 2-dim array instead for grayscale images, right？

Reply:
Right, thanks, added (H, W) shape to the docstring.